etc/kayobe/environments/ci-multinode/dns.yml

```diff
@@ -0,0 +1,8 @@
+---
+resolv_is_managed: true
+
+# List of DNS nameservers.
+resolv_nameservers:
+  - 192.168.39.4
+  - 192.168.39.5
+  - 192.168.39.6
```

Comment:
Isn't there a bootstrapping issue here? DNS will be broken until designate is deployed.

Which hosts actually need to use Designate as their DNS server?

Reply:
I'd prefer to just document how to access the Designate DNS servers. Typically controllers and hypervisors would not need to access a service deployed on the cloud. The Ansible control host might be used for manual testing, and the seed runs tempest, which could potentially test designate.